src/_gettsim/transfers/rente.py

```diff
@@ -693,14 +785,14 @@ def ges_rente_vorauss_regelrente(ges_rente_wartezeit_5: float) -> bool:
     Eligibility as bool.
 
     """
-    out = ges_rente_wartezeit_5 >= 5
+    out = ges_rente_wartezeit_5
```

Comment:
```suggestion
    return ges_rente_wartezeit_5
```

Not sure we need the function, but might be good to keep it for symmetry reasons.

Reply:
done